envs/monkey_zoo/blackbox/README.md

```diff
@@ -12,8 +12,25 @@ this information in the GCP Console `Compute Engine/VM Instances` under _Externa
 #### Running in command line
 Run the following command:
 
-`monkey\envs\monkey_zoo\blackbox>python -m pytest --island=35.207.152.72:5000 test_blackbox.py`
+`monkey\envs\monkey_zoo\blackbox>python -m pytest -s --island=35.207.152.72:5000 test_blackbox.py`
 
 #### Running in PyCharm
-Configure a PyTest configuration with the additional argument `--island=35.207.152.72` on the 
+Configure a PyTest configuration with the additional arguments `-s --island=35.207.152.72` on the 
 `monkey\envs\monkey_zoo\blackbox`.
+
+### Running telemetry performance test
+To run telemetry performance test follow these steps:
+1. Gather monkey telemetries.
+    1. Enable "Export monkey telemetries" in Configuration -> Internal -> Tests if you don't have 
+    exported telemetries already.
+    2. Run monkey and wait until infection is done.
+    3. All telemetries are gathered in `monkey/test_telems`
+2. Run telemetry performance test.
+    1. Move directory `monkey/test_telems` to `envs/monkey_zoo/blackbox/tests/performance/test_telems`
+    2. (Optional) Use `envs/monkey_zoo/blackbox/tests/performance/utils/telem_parser.py` to multiply 
+    telemetries gathered.
```

Comment:
Running this script with no files in the directory (messed up stage 2.1.) causes no error, just looks weird. Probably should raise an error to the user

Reply:
Rename script to `telem_multiplier`/`telem_fabricator`/`fabricate_telems_from_samples`

Reply:
I don't understand the reasoning behind suggested renaming. `telem_multiplier/telem_fabricator/fabricate_telems_from_samples` presumes that we have multiple multipliers and multiple fabricators, which we don't. Regarding script path in general, I'd like to keep it close to the perfromance test, as this is the only place telemetries are used. 
I'll try to improve directory structure and will segregate telemetry parsing from telemetry fabrication.

Reply:
Oops, maybe I wasn't clear - I was suggest to pick one of the options... Like, rename it to telem_multiplier OR telem_fabricator OR fabricate_telems_from_samples  

Sorry 😅😅

Reply:
Ahhh, my bad. You can keep using this syntax, I'll keep it in mind in the future